modules/ossm-rn-known-issues.adoc

```diff
@@ -19,7 +19,10 @@ These limitations exist in {ProductName}:
 
 * Graph layout - The layout for the Kiali graph can render differently, depending on your application architecture and the data to display (number of graph nodes and their interactions). Because it is difficult if not impossible to create a single layout that renders nicely for every situation, Kiali offers a choice of several different layouts. To choose a different layout, you can choose a different *Layout Schema* from the *Graph Settings* menu.
 
-* {ProductName} does not support installation on a restricted network.
+[NOTE]
```

Comment:
@neal-timpe This note should be removed, kafka is now supported as part of the Jaeger standalone release.

Reply:
removed.